src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -24,6 +24,47 @@ namespace Microsoft.PowerShell.EditorServices
     /// </summary>
     public class AnalysisService : IDisposable
     {
+        #region Static fields
+
+        /// <summary>
+        /// Defines the list of Script Analyzer rules to include by default if
+        /// no settings file is specified.
+        /// </summary>
+        private static readonly string[] s_includedRules = new string[]
+        {
+            "PSUseToExportFieldsInManifest",
+            "PSMisleadingBacktick",
+            "PSAvoidUsingCmdletAliases",
+            "PSUseApprovedVerbs",
+            "PSAvoidUsingPlainTextForPassword",
+            "PSReservedCmdletChar",
+            "PSReservedParams",
+            "PSShouldProcess",
+            "PSMissingModuleManifestField",
+            "PSAvoidDefaultValueSwitchParameter",
+            "PSUseDeclaredVarsMoreThanAssignments",
+            "PSPossibleIncorrectComparisonWithNull",
+            "PSAvoidDefaultValueForMandatoryParameter",
+            "PSPossibleIncorrectUsageOfRedirectionOperator"
+        };
+
+        /// <summary>
+        /// An empty diagnostic result to return when a script fails analysis.
+        /// </summary>
+        private static readonly PSObject[] s_emptyDiagnosticResult = new PSObject[0];
```

Comment:
Should we use `Array.Empty<PSObject>()` here?  That way the empty array will be cached (or pulled from cache)

Reply:
Looks like [it doesn't](https://docs.microsoft.com/en-us/dotnet/api/?view=netframework-4.5.1&term=Array.Empty).

Reply:
Aww :/

Probably not worth setting up then since we'll get it when we move to standard.